include/game.h

```diff
@@ -712,7 +712,8 @@ typedef struct {
     /* 0x0 */ u16 x;
     /* 0x2 */ u16 y;
     /* 0x4 */ u16 roomId;
-    /* 0x6 */ u16 unk6;
+    /* 0x6 */ u16 unk6; // Current Stage ID to reload Tile GFX if you move
```

Comment:
I think change these to currentStageId and nextStageId? I'd get a second opinion on the names before committing though.

Reply:
I think those are good names. I did not change them in my PR because then everywhere they are referenced would need changing and I wasn't wanting to name the variable on my own.